python/neuroglancer/ingest/tasks.py

```diff
@@ -129,25 +415,44 @@ def update(self, task):
         """
         raise NotImplemented
 
-    def lease(self):
+    def lease(self, tag=''):
         """
         Acquires a lease on the topmost N unowned tasks in the specified queue.
         Required query parameters: leaseSecs, numTasks
         """
-        tasks = self.api.lease(
-            project=self._project,
-            taskqueue=self._queue_name, 
-            numTasks=1, 
-            leaseSecs=600).execute()
+        if not tag:
```

Comment:
Is there ever a time you don't want to have a tag? I thought the tag determines what gets processed.